Sources/App/WebView/WebViewWindowController.swift

```diff
@@ -469,3 +471,48 @@ extension WebViewWindowController: OnboardingStateObserver {
         }
     }
 }
+
+// MARK: - Biometrics lock
+
+extension WebViewWindowController {
+    private func listenForBiometricsLockRelatedEvents() {
```

Comment:
what happens if the user gets signed out (e.g. remotely,  due to token) while this is happening?

Reply:
How can I easily reproduce that? I think the overlay will stay there, but it's nice to verify

Reply:
go into your user profile & delete the token for your device in your profile. you can do it from the same machine or other.

if the overlay stays, it may be worth killing it off since there's nothing it's protecting (if it's the last account) and may block signing into something new. there's no way to get to settings during onboarding.

Reply:
Tested. Indeed the "lock screen" was still being shown even with no servers. So I updated it in a way that the user needs to authenticate one last time after the token expires, and then the biometric lock will be disabled and only reenabled if the user sets up a new server and manually enables the toggle.